addons/l10n_cl/models/account_move.py

```diff
@@ -21,7 +21,11 @@ def _get_l10n_latam_documents_domain(self):
                 self.journal_id.l10n_latam_use_documents:
             return super()._get_l10n_latam_documents_domain()
         if self.journal_id.type == 'sale':
-            domain = [('country_id.code', '=', "CL"), ('internal_type', '!=', 'invoice_in')]
+            if self.move_type == 'out_refund':
+                internal_types_domain = ('internal_type', '=', 'credit_note')
+            else:
+                internal_types_domain = ('internal_type', '!=', 'invoice_in')
```

Comment:
@Danisan should not the credit note not be visible in the regular invoice?  
```suggestion
                internal_types_domain = ('internal_type', 'not in', ['invoice_in', 'credit_note'])
```